.github/workflows/add_issues_to_project.yml

```
@@ -1,4 +1,4 @@
-name: Adds issues to project board
+name: Assign issue/pr to project
```

v optional/minor nit, feel free to merge without
```suggestion
name: Assign issue/PR to project
```